src/query/block/information.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
super nit: Rename this file `info.go` for more "Go-ish" name?

Reply:
Ah fair enough; thought the general idea was functions and types have full names but variables have Go-ish names, will change it